packages/components/time_region_value/src/routes/[id]/[year].svelte

```diff
@@ -509,135 +471,361 @@
+									isInteractive={true}
+									key='NUTS_ID'
+									keyToColor={$doFilterRegionsStore
+										? keyToColorFiltered
+										: keyToColorAll
+									}
+									on:entered={onEnteredRegion}
+									on:exited={onExitedRegion}
+									theme={{
+										defaultFill: defaultGray,
+										defaultStroke: 'gray',
+										defaultStrokeWidth: 0.25,
+										focusedStroke: theme.colorBlack,
+										focusedStrokeWidth: 1.5,
+										selectedStroke: theme.colorBlack,
+										selectedStrokeWidth: 0.5,
+									}}
+									width={mapWidth}
+								/>
+
+								<!-- cities -->
+
+								{#if cities}
+									<g class='cities'>
+										{#each cities as {isLeft, name, X, Y, dx, dy}}
+											<g transform='translate({X},{Y})'>
+												<circle r={markerRadius}/>
+												<text
+													{dx}
+													{dy}
+													class:isLeft
+													class='background'
+													font-size={labelsFontSize}
+												>{name}</text>
+												<text
+													{dx}
+													{dy}
+													class:isLeft
+													font-size={labelsFontSize}
+												>{name}</text>
+											</g>
+										{/each}
+									</g>
+								{/if}
+
+								<!-- legend -->
+
+								<g transform='translate(0,{legendHeight})'>
+									<ColorBinsG
+										width={legendBarThickness}
+										height={legendHeight}
+										bins={colorBins}
+										flags={{
+											isVertical: true,
+											withBackground: true,
+										}}
+										theme={{
+											backgroundColor: theme.colorWhite,
+											backgroundOpacity: 0.5,
+										}}
+										ticksFormatFn={formatFn}
+									/>
+								</g>
+
+							</svg>
+
+							<!-- tooltip -->
+
+							{#if $tooltip.isVisible}
+								<div
+									class='tooltip'
+									style={$tooltip.style}
+								>
+									<header>
+										<span>{$tooltip.regionId}</span>
+										{#if $tooltip.value}
+										<span>{$tooltip.value}</span>
+										{/if}
+									</header>
+									<div>
+										<span>{$tooltip.nuts_label}</span>
+									</div>
+								</div>
+							{/if}
+
+						{/if}	<!-- {#if showMap} -->
+					</div>
+
+					<!-- barchart -->
+
+					<div class='barchart'>
+						<BarchartVDiv
+							{focusedKey}
+							{formatFn}
+							{keyToLabel}
+							{selectedKeys}
+							isInteractive={true}
+							items={$doFilterRegionsStore ? filteredItems : items}
+							keyToColor={keyToColorAll}
+							on:entered={onEnteredBar}
+							on:exited={onExitedBar}
+							refs={barchartRefs}
+							shouldResetScroll={true}
+							shouldScrollToFocusedKey={true}
+							theme={{
+								barDefaultColor: defaultGray,
+								focusedKeyColor: 'rgb(211, 238, 253)',
+								titleFontSize: '1.2rem',
+							}}
+							title={barchartTitle}
+						/>
+					</div>
+
+				{:else}
+
+					<!-- <div class='message'>
+						<span>No data</span>
+					</div> -->
+
+				{/if} <!-- filteredData.length  -->
```

Comment:
Does this need a `// TODO`?

Reply:
I'm using these to quickly see what `if` we're closing to avoid having to scroll up to the correspondent opening `{#if`